src/assets/styles/layout.scss

```diff
@@ -6,8 +6,8 @@
 $breakpoints: (
   xs: 576px,
   sm: 768px,
-  md: 1024px,
-  lg: 1200px,
+  md: 992px,
+  lg: 1248px,
   xlg: 1600px,
 );
```

Comment:
Breakpoints could be refactored in this PR or later to match with the values and names of HDS's current breakpoints (The values match but the names don't: `xs` matches `$breakpoint-s`, `sm` matches `$breakpoint-m`, `md` matches `$breakpoint-l` and `lg` matches `$breakpoint-xl` [1].

You have changed the values so the UI has also changed, have you compared how the change of md/lg breakpoint values from 1024px/1200px to 992px/1248px has affected the UI?

[1] hds-design-tokens/lib/breakpoint/breakpoint.scss from hds-design-tokens v3.7.0:
```scss
$breakpoint-xs: 320px;
$breakpoint-s: 576px;
$breakpoint-m: 768px;
$breakpoint-l: 992px;
$breakpoint-xl: 1248px;
```

Reply:
lets fix in separate PR